.github/workflows/ci.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches:
       - master
```

Comment:
We can remove this line imo, since bors will be the only one pushing to master (we should disable pushing to master in the repo settings).

Reply:
That's correct in the general case, but I think we are going to need a transition phase where merges to master occur both through PRs and manually, hence this remains useful.